### ISSUE TITLE
Option to play without guidelines

### ISSUE THREAD

**Issue by forteller** [Sothatsit/RoyalUrClient#89]
Guidelines are very nice and all, but they make the game less realistic. Normally you would not have this help, and thus might make mistakes by counting wrong before moving. This is a part of how boardgames are supposed to be, IMO. An option to turn on or off guidelines would be appreciated. Thanks!

**Comment by Sothatsit** [comment]
This would be cool! Could be added together with #33 in a settings screen.

**Comment by forteller** [comment]
I'm glad you like the idea! But I would recommend against having it in the settings page. I think it should be a check box on the new game screen, so it's easy to choose pr game how you want it. :)

**Comment by Potherca** [comment]
I think this issue is also related to #47, as "Select Game Mode" would be needed for both.

**Comment by Sothatsit** [comment]
> I think this issue is also related to #47, as "Select Game Mode" would be needed for both.

I agree, these are similar, although I think it would be good to have them as separate settings. If we add more routes, then we'd also want to be able to turn off the guidelines for each. I quite like the simplicity of the game page as it is now, although I think it may have to change to support all these new settings.
